MAUI/ImageEditor/shape.md

```diff
@@ -51,6 +51,84 @@ The [`AnnotationShape`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Imag
 * DoubleArrow
 * DottedArrow
 * DottedDoubleArrow
+* Polygon
+* Polyline
+
+## Polygon
+
+A polygon is formed by connecting a series of straight lines, and in addition to the `Points` collection includes specific points, a line is automatically connects the first and last points. 
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+   <Grid RowDefinitions="0.9*, 0.1*">
+        <imageEditor:SfImageEditor x:Name="imageEditor"
+                                   Source="image.jpeg" />
+        <Button Grid.Row="1"
+                Text="Polygon"
+                Clicked="OnPolygonClicked" />
+    </Grid>  
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    private void OnPolygonClicked(object sender, EventArgs e)
+    {
+        this.imageEditor.AddShape(AnnotationShape.Polygon,
+            new ImageEditorShapeSettings()
+            {
+                Points = new PointCollection
+                {
+                   new Point(40, 10),
+                   new Point(70, 80),
+                   new Point(10, 50)
+                },
+            });
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+![Polygon annotation in .NET Maui ImageEditor](images/shape/imageeditor-polygon-annotation.png)
+
+## Polyline
+
+It draws a series of connected straight lines and used to create triangles, pentagons, and other shapes with multiple points.
```

Comment:
It draws a series of connected straight lines. It is similar to a polygon, except the last point in a polyline is not connected to the first point.